recipe/recipe.yaml

```
@@ -32,17 +43,27 @@ requirements:
     - go-licenses
 
 tests:
+  - package_contents:
+      strict: true
+      files:
+        - bin/brev
+        - share/bash-completion/completions/brev
+        - share/zsh/site-functions/_brev
+        - share/fish/vendor_completions.d/brev.fish
+
   - script:
       - brev --version
+      - brev --help
+
 about:
   homepage: https://developer.nvidia.com/brev
   summary: 'Brev CLI'
   description: |
     CLI tool for managing workspaces provided by brev.dev
   license: MIT
-  license_file: 
+  license_file:
     - LICENSE
-    - library_licenses/**/**/LICENSE
+    - library_licenses/
```

```suggestion
    - library_licenses/**/**/LICENSE*
    - library_licenses/**/**/NOTICE*
```
I think this would make most sense for now